src/main/java/net/openhft/chronicle/threads/NotifyDiskLowLogWarn.java

```diff
@@ -0,0 +1,20 @@
+package net.openhft.chronicle.threads;
+
+import net.openhft.chronicle.core.Jvm;
+
+import java.nio.file.FileStore;
+
+public class NotifyDiskLowLogWarn implements NotifyDiskLow {
+    @Override
+    public void panic(FileStore fileStore) {
+        Jvm.warn().on(DiskSpaceMonitor.class, "your disk " + fileStore + " is almost full, " +
```

Comment:
I believe it has to be `Jvm.error()` at least. 
Also, should we consider a more aggressive behavior for this case? Like throwing an `Error` and gracefully terminating the JVM? On second thought: we should be doing such check and terminating when trying to allocate extra space.

Reply:
Agree re: `Jvm.error()`
Please see also https://github.com/ChronicleEnterprise/Chronicle-Services-Enterprise/pull/444